samples/bluetooth/shell_bt_nus/README.rst

```diff
@@ -17,31 +17,42 @@ Requirements
 
 * One of the following development boards:
```

Comment:
```suggestion
The sample supports the following development kits:
```
This is for consistency with other samples, as implemented by Uma.

Reply:
fixed